bentoml_cli/models.py

```diff
@@ -244,9 +244,15 @@ def pull(model_tag: str, force: bool):  # type: ignore (not accessed)
         default=False,
         help="Forced push to yatai even if it exists in yatai",
     )
-    def push(model_tag: str, force: bool):  # type: ignore (not accessed)
+    @click.option(
+        "-t",
+        "--threads",
+        default=10,
+        help="Number of threads to use for upload",
+    )
+    def push(model_tag: str, force: bool, threads: int = 10):  # type: ignore (not accessed)
```

Comment:
Ah I think you don't have to specify the defaults here, since you already pass it to `click.option`.
```suggestion
    def push(model_tag: str, force: bool, threads: int):  # type: ignore (not accessed)
```